### ISSUE TITLE
[bug] Bracketed term lost with :not(.hyphenated-classname)

### ISSUE THREAD

**Issue by ashsearle** [FullHuman/purgecss#24]
Reproducible with:

```css
.bar:not(.foo) { color: red; }
.bar:not(.foo-bar) { color: blue; }
```

```html
<p class="bar">anything</p>
```

Expected output CSS:
```css
.bar:not(.foo) { color: red; }
.bar:not(.foo-bar) { color: blue; }
```

Actual output CSS:
```css
.bar:not(.foo) { color: red; }
.bar:not { color: blue; }
```

```bash
$ purgecss --version
0.12.0
```

**Comment by Ffloriel** [comment]
Thank you for reporting this issue.
It should be fix in v0.13.0.
Let me know if you find any issue

**Comment by ashsearle** [comment]
That's working great now.  Thanks :-)
